specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/examples/getSoftwareUpdateProfile.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "mySubscriptionId",
+    "resourceGroupName": "myResourceGroupName",
+    "vmName": "myVMName",
+    "api-version": "2018-06-30-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/mySubscriptionId/resourceGroups/myResourceGroupName/providers/Microsoft.Compute/virtualMachines/myVMName/providers/Microsoft.GuestConfiguration/softwareUpdateProfile/default",
+        "name": "default",
+        "type": "Microsoft.GuestConfiguration/softwareUpdateProfile",
+        "properties": {
+          "computerName": "myVMName",
+          "computerIP": "40.85.217.32",
+          "computerEnvironment": "Azure",
+          "osType": "Windows",
+          "osName": "Microsoft Windows 10 Enterprise",
+          "osVersion": "10.0.17763 N/A Build 17763",
+          "osMajorVersion": "",
+          "osMinorVersion": "",
+          "osFullName": "",
+          "osArch": "x64",
+          "computerStartTime": "2018-10-30T17:31:02Z",
+          "restartPending": "True",
+          "restartPendingSince": "2018-11-01T12:34:00Z",
+          "guestConfigAgentVersion": "8.0.11081.0",
+          "guestConfigAgentStartTime": "2018-10-30T17:31:02Z",
+          "netRuntimeVersion": ".NET Framework 4.6.1",
+          "windowsUpdateSetting": "Notify before installation",
+          "windowsUpdateAgentVersion": "10.0.14393.2368",
+          "windowsUpdateIncludeRecommendedUpdates": "",
+          "wsusServer": "http://rd2-sup-hlb.redmond.corp.microsoft.com:80",
+          "linuxPackageManager": "",
+          "criticalCount": "2",
+          "securityCount": "0",
+          "otherCount": "0",
+          "approvalSource": "",
+          "latestAssessmentTimeTaken": "2018-10-30T17:31:02Z",
+          "lastUpdateApplied": "",
+          "updateAssessmentErrors": "The service cannot be started, either because it is disabled or because it has no enabled devices associated with it. (Exception from HRESULT: 0x80070422)",
```

Comment:
Mike - this is for user troubleshooting. What did you mean by "make it a little more explicit"?

Reply:
Is it intended for user self-help or for our own investigation?
Regarding "make it more explicit": A few different things. Is the UX set up to expose the data? Are we planning on making it query able via ARG? 
On a related note, this is just raw output right now that can come from a few different sources. It is fine if it is just for our own investigation, but it seems like it may be of limited use for a user to perform self-help.

Reply:
It's for user self-help. We can always make the string error message more detailed, depending on the type of exception encountered on the agent  to make it more actionable for the user.